src/StrawberryShake/Tooling/src/Configuration/VisibilityOverride.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace StrawberryShake.Tools.Configuration
+{
+    public enum VisibilityOverride
```

Comment:
If we do not look at this as an override but as a setting we could just name it what it is: `AccessModifier`

Reply:
Done.